open-source/ip.md

```diff
@@ -0,0 +1,34 @@
+# Intellectual property
+
+This page describes our approach to intellectual property with any assets we interact with regarding open source code, text, images, and upstream communities.[^1]
+
+[^1]: See [the CS&S briefing on Intellectual Property](https://www.codeforsociety.org/resources/intellectual-property-in-open-projects) for an overview of some considerations here.
+
+## Software licenses
+
+### In 2i2c repositories
+
+Our fiscal sponsor holds the rights to any code that is generated in the 2i2c repository ([`@2i2c-org`](https://github.com/2i2c-org)).
+
+Any code that we control should be released under an [OSI approved license](https://opensource.org/licenses) with permissive restrictions.
+For example, the [Apache License 2.0](https://opensource.org/licenses/Apache-2.0) or the [BSD-3 Clause "New" or "Revised"](https://opensource.org/licenses/BSD-3-Clause) license. By default, we use [the BSD 3-Clause license](https://opensource.org/licenses/BSD-3-Clause).
```

Comment:
```suggestion
For example, the [Apache License 2.0](https://opensource.org/licenses/Apache-2.0) or the [BSD-3 Clause "New" or "Revised"](https://opensource.org/licenses/BSD-3-Clause) license. By default, we use use whatever license is most prevalent in the community where we operate in. For example, in the scientific python community, that is [the BSD 3-Clause license](https://opensource.org/licenses/BSD-3-Clause).
```

Reply:
Be more explicit here about *why* 3-BSD, and where it is most appropriate.

Reply:
good idea, thanks!